src/table/src/EditableCell.js

```diff
@@ -62,7 +67,8 @@ class EditableCell extends React.PureComponent {
   }
 
   state = {
-    value: this.props.children
+    value: this.props.children,
+    isEditing: this.props.autoFocus
```

Comment:
What happens if the component has already initialized/mounted and the `autoFocus` prop changes after the fact? Should changes to the prop be propagated to state?

Reply:
I don't have a strong opinion on this, but I don't think it would be a bad thing to give the parent of `EditableCell` the ability to make the cell editable. I could imagine a scenario where it could be useful.

I'm imagining an editable table. A row is added, and is auto focused. A second row is added and takes the focus. Now the user deletes the second row. Maybe we want the first row to become focused again. I think in this case we would need to be able to use the autoFocused prop to update the `isEditing` state of the cell. Maybe the prop should be `focused` instead of `autoFocused` in this case. What do you think?

Reply:
`autoFocus` is kind of an interesting prop. It maps to `autofocus` which is a real `input` attribute and its behavior is to make the input focused upon presentation. That said I don't think it makes much sense for us to respect post-mount changes to the prop because that would seemingly deviate from the native behavior.

(I agree with your thoughts on changing the name of the prop should we change the behavior)

Thoughts?

Reply:
Yeah, originally I chose the name `autoFocus` to resemble the html input `autofocus` but you're right that this behaves differently. 

I'm fine with either a) changing the prop name to `focused` or b) changing the implementation so that the `autoFocus` prop is only used to seed the `isEditing` state, and is never used again. 

I think I'll do **b** because it's functionality we know we need now, without speculation about what we might need in the future.

Reply:
Agreed!